seeds/quickbooks__balance_sheet_account_class_order.csv

```diff
@@ -0,0 +1,4 @@
+account_class,ordinal
+Asset,1
+Liability,2
+Equity,3
```

Comment:
## Love this idea!
I like this idea; however, I feel we can make this a bit more robust and dual purpose. 

### Robust
I imagine customers will want to have more control over the ordering of the account type and subtypes possibly. The class is a great high level start, but I imagine for the P&L they will want to have cost of revenue shown higher than other operating expenses. It would be great if we could provide that level of configurability for customers. I think the format you have is great, we can maybe expand on it a bit more.

### Dual Purpose
What if we combined both the Balance Sheet and P&L seeds into one `quickbooks__financial_statement_ordinal` (or something of the sort (lol ordinal pun)). My thoughts is it would look something like the following:

| **report**  | **account_number**  |  **account_class**  | **account_type** | **account_sub_type** |  **ordinal** |
| ------- | ----------------------- | --------- | --------- | --------- | --------- |
| Balance Sheet | null | Asset | Other Asset | Goodwill | 1 |
| Profit and Loss | 555 | null | null | null | 1 |

My idea would then be the join in the final models could then be easily adjusted to coalesce based on the hierarchy going from finest grain to highest. This way customers can configure how much or as little as they want. Our default however could be the standard class level you have included in this PR.
```sql
select
    *,
     coalesce(account_number_ordinal.ordinal, account_sub_type.ordinal, account_type.ordinal, account_class.ordinal) as ordinal

from general_ledger_by_period
 left join ordinals as account_number_ordinal
        on general_ledger_by_period.account_number = account_number_ordinal.account_number
 left join ordinals as account_class_ordinal
        on general_ledger_by_period.account_class = account_class_ordinal.account_class
 left join ordinals as account_type_ordinal
        on general_ledger_by_period.account_type = account_type_ordinal.account_type
 left join ordinals as account_sub_type
        on general_ledger_by_period.account_sub_type = account_sub_type.account_sub_type
```

Curious to get your thoughts on this!

Reply:
This makes sense to me! I've consolidated the seed files now.